lib/axon/schedules.ex

```diff
@@ -88,7 +88,7 @@ defmodule Axon.Schedules do
   @doc ~S"""
   Cosine decay schedule.
 
-  $$\gamma(t) = \gamma_0 * (1 - \alpha)*(\frac{1}{2}(1 + \cos{\pi \frac{t}{k}})) + \alpha$$
+  $$\gamma(t) = \gamma_0 * \left(\frac{1}{2}(1 - \alpha)(1 + \cos\pi \frac{t}{k}) + \alpha\right)$$
```

Comment:
This one seems to correctly reflect the implemented code. @seanmor5 could you check if the code itself isn't what's wrong?

Reply:
I compared with the paper as well as the Optax implementation and I _think_ the code's correct.